tests/fmt.rs

```diff
@@ -1010,6 +1010,36 @@ mod table {
         let p = Parser::new_ext(&generated_markdown, Options::all());
         let generated_events: Vec<_> = p.into_iter().collect();
 
+        assert_eq!(original_events, generated_events);
+    }
+    #[test]
+    fn table_with_pipe_in_column() {
+        use pulldown_cmark::{Options, Parser};
+
+        let original_table_markdown = indoc!(
+            r"
+            | \| | a\|b |
+            |----|------|
+            | \| | a\|b |"
+        );
+        let p = Parser::new_ext(original_table_markdown, Options::all());
+        let original_events: Vec<_> = p.into_iter().collect();
+
+        let (generated_markdown, _) = fmte(&original_events);
+
+        assert_eq!(
+            generated_markdown,
+            indoc!(
+                r"
+                |\||a\|b|
+                |-|---|
+                |\||a\|b|"
+            )
+        );
+
+        let p = Parser::new_ext(&generated_markdown, Options::all());
+        let generated_events: Vec<_> = p.into_iter().collect();
```

Comment:
That's interesting, I thought there already is a utility for round-tripping. If not, or if this is a specific variety, maybe an existing one can be adjusted or a new one can be added.

Reply:
In these cases, I add a function that has an additional parameter, and call it from the 'previous' one.